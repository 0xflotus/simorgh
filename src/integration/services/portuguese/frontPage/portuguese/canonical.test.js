/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service portuguese
 * @pathname /portuguese
 */

import runCanonicalTests from '../../../../pages/frontPage/canonicalTests';

describe('Canonical portuguese frontPage', runCanonicalTests);
