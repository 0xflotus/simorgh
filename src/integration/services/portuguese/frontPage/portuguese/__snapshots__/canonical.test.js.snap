// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical portuguese frontPage A11y I can see the skip to content link 1`] = `"Vá para o conteúdo"`;

exports[`Canonical portuguese frontPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=33&amp;p=portuguese.page&amp;x1=[urn:bbc:cps:30d476fb-7a8c-d0ad-e050-380a4e801c9d]&amp;x2=[responsive]&amp;x3=[news-portuguese]&amp;x4=[pt-BR]&amp;x7=[index-home]&amp;x8=[simorgh-nojs]&amp;x9=[Notícias,+vídeos,+análise+e+contexto+em+português+-+BBC+News+Brasil]&amp;x11=[1970-01-17T22:01:20.407Z]&amp;x12=[1970-01-19T02:08:51.140Z]\\"/>"`;

exports[`Canonical portuguese frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/portuguese"`;

exports[`Canonical portuguese frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. A BBC não se responsabiliza pelo conteúdo de sites externos. Leia sobre nossa política em relação a links externos."`;

exports[`Canonical portuguese frontPage Header I can see the branding 1`] = `"BBC News, Brasil"`;

exports[`Canonical portuguese frontPage I can see Index Alsos 1`] = `"Histórias relacionadas"`;

exports[`Canonical portuguese frontPage I can see sections 1`] = `"Top story - Brasil"`;

exports[`Canonical portuguese frontPage I can see sections 2`] = `"VídeoVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 3`] = `"Brasil"`;

exports[`Canonical portuguese frontPage I can see sections 4`] = `"Destaque"`;

exports[`Canonical portuguese frontPage I can see sections 5`] = `"Curiosidades"`;

exports[`Canonical portuguese frontPage I can see sections 6`] = `"EconomiaVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 7`] = `"CiênciaVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 8`] = `"Meio AmbienteVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 9`] = `"HistóriaVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 10`] = `"SaúdeVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 11`] = `"SociedadeVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 12`] = `"Aprenda inglêsVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 13`] = `"Destaque"`;

exports[`Canonical portuguese frontPage I can see sections 14`] = `"Especiais e GaleriasVer todos"`;

exports[`Canonical portuguese frontPage I can see sections 15`] = `"Destaques e Análises Ver todos"`;

exports[`Canonical portuguese frontPage I can see the headline 1`] = `"BBC News, Brasil - Início"`;

exports[`Canonical portuguese frontPage SEO Canonical link 1`] = `"http://localhost:7080/portuguese"`;

exports[`Canonical portuguese frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical portuguese frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical portuguese frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical portuguese frontPage SEO Heading level 1 1`] = `"BBC News, Brasil - Início"`;

exports[`Canonical portuguese frontPage SEO Lang attribute 1`] = `"pt-BR"`;

exports[`Canonical portuguese frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/portuguese",
    "@type": "WebPage",
    "name": "Notícias, vídeos, análise e contexto em português",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png",
      "width": 1024,
    },
    "name": "BBC News Brasil",
    "publishingPrinciples": "https://www.bbc.com/portuguese/institutional-50054434",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png",
  "url": "http://localhost:7080/portuguese",
}
`;

exports[`Canonical portuguese frontPage SEO OG description 1`] = `"Na BBC Brasil você fica por dentro do que acontece no Brasil e no mundo. Com redações em Londres e São Paulo e mais de 200 repórteres internacionais, nossa cobertura leva a você as principais notícias de política, economia, ciência, saúde e comportamento."`;

exports[`Canonical portuguese frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png"`;

exports[`Canonical portuguese frontPage SEO OG image alt 1`] = `"BBC News Brasil"`;

exports[`Canonical portuguese frontPage SEO OG locale 1`] = `"pt-BR"`;

exports[`Canonical portuguese frontPage SEO OG site name 1`] = `"BBC News Brasil"`;

exports[`Canonical portuguese frontPage SEO OG title 1`] = `"Notícias, vídeos, análise e contexto em português - BBC News Brasil"`;

exports[`Canonical portuguese frontPage SEO OG type 1`] = `"website"`;

exports[`Canonical portuguese frontPage SEO OG url 1`] = `"http://localhost:7080/portuguese"`;

exports[`Canonical portuguese frontPage SEO Page title 1`] = `"Notícias, vídeos, análise e contexto em português - BBC News Brasil"`;

exports[`Canonical portuguese frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical portuguese frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical portuguese frontPage SEO Twitter creator 1`] = `"@bbcbrasil"`;

exports[`Canonical portuguese frontPage SEO Twitter description 1`] = `"Na BBC Brasil você fica por dentro do que acontece no Brasil e no mundo. Com redações em Londres e São Paulo e mais de 200 repórteres internacionais, nossa cobertura leva a você as principais notícias de política, economia, ciência, saúde e comportamento."`;

exports[`Canonical portuguese frontPage SEO Twitter image alt 1`] = `"BBC News Brasil"`;

exports[`Canonical portuguese frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png"`;

exports[`Canonical portuguese frontPage SEO Twitter site 1`] = `"@bbcbrasil"`;

exports[`Canonical portuguese frontPage SEO Twitter title 1`] = `"Notícias, vídeos, análise e contexto em português - BBC News Brasil"`;

exports[`Canonical portuguese frontPage Story Promo I can see a headline 1`] = `"Reforma da Previdência : voto na Câmara se alonga e pode acabar só em agosto"`;

exports[`Canonical portuguese frontPage Story Promo I can see an image 1`] = `
<img
  alt="Sessão para continuação da votação da PEC 6/2019 - Reforma da Previdência"
  class="StyledImg-sc-7vx2mr-0 hIxkbt"
  sizes="(max-width: 600px) 100vw, (max-width: 1008px) 50vw, 496px"
  src="https://ichef.bbci.co.uk/news/660/cpsprodpb/0554/production/_107846310_previdencia.jpg"
  srcset="https://ichef.bbci.co.uk/news/70/cpsprodpb/0554/production/_107846310_previdencia.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsprodpb/0554/production/_107846310_previdencia.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsprodpb/0554/production/_107846310_previdencia.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsprodpb/0554/production/_107846310_previdencia.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsprodpb/0554/production/_107846310_previdencia.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsprodpb/0554/production/_107846310_previdencia.jpg 320w, https://ichef.bbci.co.uk/news/660/cpsprodpb/0554/production/_107846310_previdencia.jpg 660w"
  width="976"
/>
`;
